usagov/usagov-2021
# Usagov 1127 accordion adjustments

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1127

## Description

Change all sections with the header at the top and the content at the bottom for consistency. The space between sections needs to be increased to 2.2em. 

**Testing:**

1. Go to this page: http://localhost/agency-index 
2. Check that the space between sections (website, phone number, etc.) is consistent and that the margin between them is 2.2em.
3. Check that the header of each section is at the top and the content at the bottom.
4. Try on different browsers and devices like: mobile, tablet and desktop.

## Checklist for the Developer



## Checklist for the Peer Reviewers
